action.yaml

```diff
@@ -164,14 +164,29 @@ runs:
         if [ "${{ inputs.verbose }}" == "true" ]; then
           extraArgs+=("--verbose")
         fi
+        touch /tmp/console.log
         sudo /bin/lvh run --host-mount=${{ inputs.host-mount }} --image /_images/${{ inputs.test-name }}.qcow2 \
             --daemonize -p ${{ inputs.ssh-port }}:22 --serial-port ${{ inputs.serial-port }} \
             --cpu=${{ inputs.cpu }} --mem=${{ inputs.mem }} --cpu-kind ${{ inputs.cpu-kind }} \
+            --console-log-file /tmp/console.log \
             ${extraArgs[@]}
 
     - name: Wait for VM's SSH Server
       shell: bash
       run: |
+        n=0
+        until [ "$n" -ge 30 ]; do
+          started=1
+          if grep -E ".*OK.*Started.*ssh.*" $1; then
```

Comment:
i assume we should replace `$1` with `/tmp/console.log` ? 

(Or I'm missing something and this really gets passed in as argument :roll_eyes: )

Reply:
Ah good catch thank you! I was testing this with a bash script and used `$1` so I could change the test file I was using.